package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "webpack-project",
+    "version": "1.0.0",
+    "description": "",
+    "private": true,
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "webpack serve --open",
+        "build": "webpack"
```

Comment:
- [ ] On this part please add `"dev": "webpack --mode development",`. This is to run the project in development mode. 🐶